setup/FSharp.SDK/FSharp.SDK.wixproj

```diff
@@ -10,7 +10,8 @@
   <Import Project="$(SetupRootFolder)\FSharp.Setup.props" />
 
   <PropertyGroup>
-    <OutputName>Microsoft.FSharp.SDK</OutputName>
+    <OutputName Condition="'$(IsLangPack)' == 'false'">Microsoft.FSharp.SDK.Core</OutputName>
+    <OutputName Condition="'$(IsLangPack)' == 'true'">Microsoft.FSharp.SDK.Resources.$(LocaleCode)</OutputName>
```

Comment:
Maybe do `Condition="'$(IsLangPack)' == 'true'"` and `Condition="'$(IsLangPack)' != 'true'"` (or is it `<> true`?) to prevent the case where `$(LangPack)` is set to "`True`" or "`False`" and neither of these conditions takes.